src/EditorFeatures/TestUtilities/TestExtensionErrorHandler.cs

```diff
@@ -30,7 +31,14 @@ public void HandleError(object sender, Exception exception)
                 return;
             }
 
-            ExceptionUtilities.FailFast(exception);
+            RaiseFailFastException(IntPtr.Zero, IntPtr.Zero, 0);
+
+            // the RaiseFailFastException above may have not actually killed the process if a
+            // debugger was attached. Fall back to the CLR FailFast.
+            Environment.FailFast("FailFast was issued from devenv.exe.  A crash dump should be available.");
         }
+
+        [DllImport("kernel32")]
+        static extern void RaiseFailFastException(IntPtr exceptionRecord, IntPtr contextRecord, uint flags);
```

Comment:
can we make ths cahnge in a separate PR?  seems not required for this particular piece of work.

Reply:
It is required - this removes an ambiguous definition of `ExceptionUtilities` which conflicts with another implementation of `ExceptionUtilities` that I need to use.

Reply:
Can we resolve the conflict in some other way? This implies we're just copy/pasting code then?

Reply:
No, this is only a test helper code.